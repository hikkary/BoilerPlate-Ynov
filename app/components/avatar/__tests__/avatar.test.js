import React from 'react'
import renderer from 'react-test-renderer'

import Avatar from '../'

it('renders successfully ', () => {
  const rendered = renderer.create(<Avatar />).toJSON()
  console.log(rendered)
  console.log(rendered.props.style)
  expect(rendered).toBeTruthy()
}) // test de base
// allow us to see if refacto or code change does not make crash the component

it('renders successfully ', () => {
  const rendered = renderer.create(<Avatar />).toJSON()
  console.log(rendered)
  console.log(rendered.props.style)
  expect(rendered).toMatchSnapshot()
}) // test cree snapshot pour voir les changement

it('has a red backgroundColor', () => {
  const rendered = renderer.create(<Avatar />).toJSON()
  console.log(rendered.props.style)
  expect(rendered.props.style[0].backgroundColor).toBe('red')
})
