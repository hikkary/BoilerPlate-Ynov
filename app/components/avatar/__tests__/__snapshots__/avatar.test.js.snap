// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders successfully  1`] = `
<View>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "red",
          "height": 40,
          "width": 40,
        },
      ]
    }
  />
</View>
`;
